### PR TITLE
add support for Python 3.5 infix matrix multiplication

### PR DESCRIPTION
**Pull request by zackmdavis**
This came up while working on a momentarily forthcoming pull request (the topic of which can be easily inferred) for Hy.

### CODE REVIEW

**Comment by zackmdavis** [comment]
CI informs me that this project is supposed to be Python 2.6&ndash;compatible, where the `@unittest.skipIf` decorator I had tried to use doesn't exist. Fixed-up and force-pushed.

File: tests/test_misc.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import sys
+
+import astor
+
+
+class GetSymbolTestCase(unittest.TestCase):
+
+    def test_get_mat_mult(self):
+        if sys.version_info < (2, 7):
+            # We can't use `@unittest.skipIf` or `raise skipTest` in
```
Comment (berkerpeksag) [inline tests/test_misc.py]: We are using unittest2 in Python 2.6. If you import unittest2 conditionally, you can use `unittest.skipIf` in Python 2.6:

```py
try:
    import unittest2 as unittest
except ImportError:
    import unittest
```

**Comment by berkerpeksag** [comment]
Could you also add a test to test_codegen.py?

**Comment by zackmdavis** [comment]
Hm. f27c059 uses `unittest2` if available (and also adds a test to `test_codegen`), but CI fails again on Python 2.6 (`'module' object has no attribute 'skipIf'`).

**Comment by berkerpeksag** [comment]
There's a typo in the import statement :)

```py
import unitttest2 as unittest
```

See three t in `unittest2`.

**Comment by zackmdavis** [comment]
(force-pushed) Okay, what about  30b310d? In addition to using the correct number of `t`s in "unittest2`, it explicitly checks for for `SkipTest`, and avoids the context-manager form of `assertRaises`, so it should work on Python 2.6 regardless.

**Comment by berkerpeksag** [comment]
Thanks! The test looks good. We can simply the other test a bit:

```py
@unittest.skipUnless(sys.version_info >= (3, 5), "ast.MatMult introduced in Python 3.5")
def test_get_mat_mult(self):
    self.assertEqual('@', astor.misc.get_binop(ast.MatMult()))
```

**Comment by zackmdavis** [comment]
(amended again)

**Comment by berkerpeksag** [comment]
Thanks! :) I will release 0.5 today or tomorrow.

**Comment by berkerpeksag** [comment]
Done: https://pypi.python.org/pypi/astor

**Pull request merged**